client/components/accordion/docs/example.jsx

```diff
@@ -13,12 +13,10 @@ import React from 'react';
 import Accordion from 'components/accordion';
 import Gridicon from 'components/gridicon';
 
-Accordion.displayName = 'Accordion';
-Gridicon.displayName = 'Gridicon';
-
-function AccordionExample( props ) {
+export default function AccordionExample( props ) {
 	return props.exampleCode;
 }
+AccordionExample.displayName = 'AccordionExample';
```

Comment:
@jsnajdr This is now redundant because the function name can be inferred?

Reply:
Nevermind, I misinterpreted some of your changes. I think we'll still need these for wpcalypso production env with mangled names.